cocos/core/geometry/ray.ts

```diff
@@ -149,7 +153,7 @@ export class Ray {
         return this._type;
     }
 
-    protected readonly _type: number;
+    protected readonly _type: number = enums.SHAPE_RAY;
```

Comment:
Why not use declare?

Reply:
Only use `declare` when properties are initialized in `constructor`. Otherwise, it doesn't need to use `declare` keyword.

Reply:
It is initialized in constructor.

Reply:
![WeChat5b85dd6fd0256c9a9e3ec2d733726d6b](https://github.com/user-attachments/assets/2ffea40f-7093-4e33-aed8-f24168a4e0a2)